packages/nextjs/app/builders/0x0C4D0953B769637DE90CC3Fe42f6f356543C5a45/page.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import { FaGithub, FaLinkedin } from "react-icons/fa";
+
+const BioPage: NextPage = () => {
+  return (
+    <div className="max-w-2xl mx-auto p-6 bg-white dark:bg-gray-800 rounded-lg shadow-md">
+      <div className="flex items-center mb-6">
+        <Image src="/ntk.webp" alt="Nick Kasten" width={128} height={128} className="rounded-full mr-4" />
+        <h1 className="text-5xl font-bold text-gray-800 dark:text-white">Nick Kasten</h1>
+      </div>
+
+      <div className="mb-6">
+        <pre className="font-semibold">0x0c4d0953b769637de90cc3fe42f6f356543c5a45</pre>
```

Comment:
There's an SE2 component called `<Address>` that might fit well to present this information.